flash_attn/cute/flash_fwd_sm100.py

```diff
@@ -1125,30 +1145,48 @@ def load(
                 K_or_V="V",
             )
 
-            n_block_min, n_block_max = block_info.get_n_block_min_max(seqlen, m_block)
-            load_Q(block=self.q_stage * m_block + 0, stage=0)  # Q0
-            page_idx = (
-                mPageTable[batch_idx, n_block_max - 1]
-                if const_expr(mPageTable is not None)
-                else None
-            )
-            load_K(block=n_block_max - 1, producer_state=kv_producer_state, page_idx=page_idx)  # K0
-            kv_producer_state.advance()
-            if const_expr(self.q_stage == 2):
-                load_Q(block=self.q_stage * m_block + 1, stage=1)  # Q1
-            q_producer_phase ^= 1
-            load_V(block=n_block_max - 1, producer_state=kv_producer_state, page_idx=page_idx)  # V0
-            kv_producer_state.advance()
-            for i in cutlass.range(n_block_max - 1 - n_block_min, unroll=1):
-                n_block = n_block_max - 2 - i
+            if const_expr(self.use_block_sparsity):
```

Comment:
I hate that github renders it this way, ill change to ` not use` so its easier to review